Packs/CortexXDR/Playbooks/Cortex_XDR_-_Large_Upload.yml

```diff
@@ -0,0 +1,4175 @@
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: 6581026b-5e6e-46cb-836a-1d850269f0ed
+    type: title
+    task:
+      id: 6581026b-5e6e-46cb-836a-1d850269f0ed
+      version: -1
+      name: Check Previous Similar Incidents
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "4"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 2410,
+          "y": 100
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 2864f683-d804-43bb-8946-4d90039fc751
+    type: condition
+    task:
+      id: 2864f683-d804-43bb-8946-4d90039fc751
+      version: -1
+      name: Found Results?
+      description: Determine if previous false positive incidents have been detected with similar characteristics.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "113"
+      "yes":
+      - "5"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: DBotFindSimilarIncidents
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+      - - operator: containsGeneral
+          left:
+            value:
+              complex:
+                root: DBotFindSimilarIncidents.similarIncident
+                accessor: closeReason
+                transformers:
+                - operator: uniq
+            iscontext: true
+          right:
+            value:
+              simple: Fale Positive
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 2410,
+          "y": 390
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: b1087a38-bd78-4643-8987-7d69672c538e
+    type: regular
+    task:
+      id: b1087a38-bd78-4643-8987-7d69672c538e
+      version: -1
+      name: Get Similar False-Positives Closed Incidents
+      description: Find past similar incidents based on incident fields' similarity. Includes an option to also display indicators similarity.
+      scriptName: DBotFindSimilarIncidents
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      fieldExactMatch:
+        simple: incident.type
+      fieldsToDisplay:
+        simple: closeReason,closeNotes
+      minimunIncidentSimilarity:
+        simple: "0.8"
```

Comment:
Customers and CS say most large upload alerts are false positives. We wanted to make sure the current large upload alert does not repeat the false positives triggered before and closed. Because we only want similar large upload alerts, I set the automation input 'minimunIncidentSimilarity' to 0.8. Customers should not configure this input alone, since setting it below 0.8 will result in different alerts, and setting it above 0.8 may miss some similar alerts.